### ISSUE TITLE
PeoplePicker selected suggestion background color contrast ratio is less than 3:1

### ISSUE THREAD

**Issue by qqli1209** [microsoft/fluentui#11576]
### Environment Information

- **Package version(s)**: all
- **Browser and OS versions**: all

### Describe the issue:
PeoplePicker selected suggestion backgroundColor color contrast ratio is only 1.2:1, and it's less than 3:1. 

#### Actual behavior:

PeoplePicker selected suggestion backgroundColor color contrast ratio is only 1.2:1.
![image](https://user-images.githubusercontent.com/13404320/71570491-395ba580-2b10-11ea-99f7-8fc9036ba8d3.png)

Even when **hover** on the selected suggestion, the backgroundColor color contrast ratio is only 1.7:1.
![image](https://user-images.githubusercontent.com/13404320/71570564-d6b6d980-2b10-11ea-98f8-2bb6a83752c0.png)

#### Expected behavior:

PeoplePicker selected suggestion backgroundColor color contrast ratio should be at least 3:1. We need to update the background color of selected item in [SuggestionsItem.styles.ts](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/pickers/Suggestions/SuggestionsItem.styles.ts#L70).

### Documentation describing expected behavior

This does not meet the requirement of Microsoft Accessibility Standard, see [here](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7B6CA3BF14-2635-434A-832C-28DD9A18B9FC%7D&file=MAS%201.4.11%20%E2%80%93%20Non-text%20Contrast.docx&action=default&mobileredirect=true&cid=bd4aad4d-fb4e-4e70-b502-c603a27eeeb8).

**Comment by jdhuntington** [comment]
Hi @qqli1209 - Could you clarify which elements you are comparing? When I look at the name on the background, I see `rgb(50,49,48)` on top of `rgb(237,235,233)`, which has a contrast ratio greater than `10`. When hovered, it drops to `9.66`, but is still well above `3`.

**Comment by qqli1209** [comment]
@jdhuntington Thanks for the quick response!

I mean the contrast ratio between the selected suggestion background `rgb(237,235,233)` and unselected background `rgb(255,255,255)`.

**Comment by jdhuntington** [comment]
Thanks for the clarification. This makes sense, will figure out the appropriate action to take.

**Comment by jdhuntington** [comment]
@betrue-final-final Are you able to comment on the contrast of this control's background color for indicated items? Is there another neutral color in the palette we can shift to?

**Comment by qqli1209** [comment]
Hi @betrue-final-final  Could you please help to take a look?

**Comment by betrue-final-final** [comment]
Actually the problem is keyboard focus style, which uses the same style as hover. It should be a dark grey border like dropdown
![image](https://user-images.githubusercontent.com/20363037/74367083-1b779680-4d86-11ea-9102-bac4c5c5220b.png)

**Comment by betrue-final-final** [comment]
Selection is not an issue as when the menu is clicked, the item populates the well.
